src/ep.config.json

```diff
@@ -1,8 +1,30 @@
 {
   "cortexApi":{
-    "path":"http://localhost:8080/cortex",
-    "scope":"vestri",
-    "pathForProxy":"http://localhost:9080"
+    "path":"http://localhost:7080/cortex",
```

Comment:
Was it intentional to change the port used for the store and proxy? Did this get pushed by accident?

Reply:
Pushed by accident :)